app/split_integer.py

```diff
@@ -1,4 +1,7 @@
 def split_integer(value: int, number_of_parts: int) -> list:
+    if value < number_of_parts:
+        return [1] * value + [0] * (number_of_parts - value)
```

Comment:
I think you don't need add this validate value, because you have a condition. Remove it
![image](https://github.com/user-attachments/assets/ae99f988-281f-4f38-82be-852db9f5adc7)